### PR TITLE
fix(curriculum): Extended description 

### LINKED ISSUE #47165: Adding explanation for why campers are doing the instructions
### Describe the Issue

In some steps there are description that carry instructions only, this will make campers think of why they are implementing instruction, instead of how to add these instruction and concepts into their future projects.

### Example

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-flexbox-by-building-a-photo-gallery/step-5

### Additional context

We don't have to explain the concept, but giving something small that they can research, is better than giving nothing

### PR DESCRIPTION
**Pull request by hanswang123456**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47165

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-5
 
 # --description--
 
-Below your `header`, create a new `div` element and assign a class of `gallery` to it.
+Below your `header`, create a new `div` element and assign a class of `gallery` to it. The class of 'gallery' is an identifer that will be referenced to style the image gallery and make it responsive.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md:10]: I think simpler is better:
```suggestion
Below your `header` element, create a new `div` element with the `id` of `"gallery"`. This `div` will act as a container for the gallery images.
```

**Comment by hanswang123456** [comment]
I have updated the description @jeremylt.

**Review by jeremylt — APPROVED** [review]
LGTM

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md (hunk 1 above)
Comment (Sboonny) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md]: I am worried about `"gallery"`, people may type `<div id=""gallery"">`, what do you think of removing the quotes? @jeremyIt

Comment (jeremylt) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md]: We have done ""s around string literals in other challenges. We should pick one and stick to it.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**